docs/images.md

```diff
@@ -173,6 +173,24 @@ In some cases you might only want to display an image if it is already in the lo
 
 See [CameraRoll](https://github.com/react-native-community/react-native-cameraroll) for an example of using local resources that are outside of `Images.xcassets`.
 
+### Drawable resources
+
+Android supports loading [drawable resources](https://developer.android.com/guide/topics/resources/drawable-resource) via the `xml` file type. This means you can use [vector drawables](https://developer.android.com/develop/ui/views/graphics/vector-drawable-resources) for rendering icons or [shape drawables](https://developer.android.com/guide/topics/resources/drawable-resource#Shape) for, well, drawing shapes! You can import and use these resource types the same as any other static resource or hybrid resource. Like other hybrid app resources, you have to specify image dimensions manually.
+
+```tsx
+<Image
+  source={require('./img/my_icon.xml')}
+  style={{width: 40, height: 40}}
+/>
+// or as a hybrid resource
```

Comment:
When read this out of context, it wasn't immediately clear what was `hybrid resource` referring to.

Then I read the section above (`Images From Hybrid App's Resources`) and it started to make sense.

I'm wondering, could we make the section more self-contained by emphasizing the 2 major cases:

```
For static drawables, use the `require` syntax:

<Image
  source={require('./img/my_icon.xml')}
  style={{width: 40, height: 40}}
/>

For drawables included in the Xcode asset catalogs or in the Android drawable folder, use the resource name without the extension:

<Image
  source={require({uri: 'my_icon')}
  style={{width: 40, height: 40}}
/>
```

Reply:
I split apart both code blocks into separate chunks and added these hints. I've also linked out to the appropriate section in this documentation where you can learn more about static and hybrid resources. Thanks for the suggestions!